src/kili/services/plugins/upload.py

```diff
@@ -156,6 +182,9 @@ def _retrieve_script(self) -> Path:
         if not check_file_is_py(file_path, self.verbose):
             raise ValueError("Wrong file format.")
 
+        if not check_file_contain_handler(self.plugin_name, str(file_path)):
+            raise ValueError("Do not contain PluginHandler Class. ")
+
```

Comment:
Can you do that in `_parse_script` instead?

Reply:
yes will do